src/pages/frontpage/NvaDescriptionSection.tsx

```diff
@@ -16,8 +19,16 @@ const iconStyle = { height: '2rem', width: '2rem' };
 export const NvaDescriptionSection = () => {
   const { t } = useTranslation();
 
+  const registrationQuery = useRegistrationSearch({
+    params: { aggregation: 'all' },
+  });
+
+  const personQuery = useSearchForPerson({});
+
+  const projectsQuery = useFetchProjects('', true);
```

Comment:
Syns sånne ting fort blir så mystiske, da jeg ikke vet hva `''` og `true` er for noe, så må man sjekke hooken for å vite hva som skjer. Ser denne hooken bare brukes en plass så kanskje like greit å rette det opp nå til å bli feks:
```
interface FetchProjectConfig {
  enabled?: boolean;
}

export const useFetchProjects = ({ enabled = true }: FetchProjectConfig) => {
  const { t } = useTranslation();

  return useQuery({
    enabled,
    (...)
  });
};
```

da har man kontroll over hva som skjer der man bruker hooken, siden man eksplisitt må oppgi `enabled: true|false` om man vil overstyre noe